runtime/wasm/src/mapping.rs

```diff
@@ -8,9 +8,20 @@ use std::collections::BTreeMap;
 use std::sync::Arc;
 use std::thread;
 
+const ONE_MEGA_BYTE: usize = 1 << 20; // 1_048_576
+
 lazy_static! {
     /// Verbose logging of mapping inputs
     pub static ref LOG_TRIGGER_DATA: bool = std::env::var("GRAPH_LOG_TRIGGER_DATA").is_ok();
+
+    /// Maximum stack size for the WASM runtime
+    pub static ref MAX_STACK_SIZE: usize = std::env::var("GRAPH_RUNTIME_MAX_STACK_SIZE")
+        .ok()
+        .and_then(|max_stack_size| max_stack_size.parse().ok())
+        // 1 MiB is wasmtime default
+        // https://docs.rs/wasmtime/0.27.0/wasmtime/struct.Config.html#method.max_wasm_stack
+        // https://docs.rs/wasmtime/0.27.0/src/wasmtime/config.rs.html#419
+        .unwrap_or(ONE_MEGA_BYTE);
```

Comment:
Lets default to 512 MiB since that's what we intend to try on hosted.

Reply:
Is it safe for us to change the default for **everyone**?

Reply:
We discussed on slack that we'll test on the hosted service first (before doing another release).